### DISCUSSION TITLE
Select system python

### DISCUSSION THREAD

**Discussion by s0undt3ch** [community/community#27182] [Actions]
<p>I have a project that I need to run against the system python.<br>
The ubuntu image, <code>runs-on</code> apparently has the default python set to a python actions python instead of the system python.</p>
<p>How can I make sure I’m running against the system python and not a hosted tools python?</p>
<p>Please note that I’m not setting up any python version with python actions, ie, <code>uses: actions/setup-python</code></p>

**Comment by FeliciaXmL** [comment]
<p><a class="mention" href="/s0undt3ch">@s0undt3ch</a></p>
<p>Ubuntu in Linux</p>
<p>Works well in mostly all platforms or environments.</p>

**Comment by s0undt3ch** [comment, marked as answer]
<p>Turns out I was indeed setting up python in another step.</p>

**Comment by FeliciaXmL** [comment]
<p>thanks for the support… your suggestion did it … python👍</p>
